runtime/tests/checker/utils.go

```diff
@@ -47,16 +48,23 @@ func ParseAndCheckWithOptions(
 	options ParseAndCheckOptions,
 ) (*sema.Checker, error) {
 
+	if options.Location == nil {
+		options.Location = utils.TestLocation
+	}
+
 	program, err := parser2.ParseProgram(code)
 	if !options.IgnoreParseError && !assert.NoError(t, err) {
```

Comment:
Double negatives are confusing, could this be replaced by `assert.Error(t, err)`.

Reply:
Agreed, but here we want to make sure that the assertion succeeds that there is no error – if it does not succeed, then an unexpected error occurred and we report it. 
Asserting there is an error is not the same.